bin/mac-uninstall-brew.sh

```diff
@@ -47,5 +47,10 @@ echo "Removing other things that may upgrade if user reinstalls..."
 sudo rm -rf ~/Library/Caches/pip
 sudo rm -rf ~/.npm
 sudo rm -rf ~/.yarnrc
+# NOTE(john): We could delete the pnpm store, however pnpm's store isn't
+# just a cache, it's all the packages installed by pnpm. It symlinks to
+# the global store, so deleting it would require reinstalling all the
+# packages. If we do want to delete it, I think this would work:
+# sudo rm -rf "$(pnpm store path)"
```

Comment:
I'd delete it if it were me.  But I don't know that it's a big deal.

On the other hand, `make allclean` (in webapp) should probably delete this!  And you may want to clear it out for `make fix_deps` as well.  Not this PR, obviously.

Reply:
Hmm... so I'm not sure that `allclean` should delete this. This is a system-wide thing. If this gets deleted them you'll have `node_modules` in other repos that are now symlinked to files that no longer exist. The problem is that it's not just a cache, it's the source-of-truth for the NPM packages and those packages are symlinked to this location, so things could get weirdly broken very quickly. I suspect I want to err on the side of not deleting, see what happens, and if that's more complicated than the alternative, we can reconsider.

Reply:
OK, I'll leave it to you to do what's reasonable.  But "allclean" is trying to clean all the (webapp) things!  Maybe there's a middle ground.